client/my-sites/checkout/src/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -64,6 +64,9 @@ export default function useCreatePaymentCompleteCallback( {
 	siteSlug,
 	sitelessCheckoutType,
 	checkoutFlow,
+	connectAfterCheckout,
+	adminUrl: wpAdminUrl,
```

Comment:
does wpAdminUrl need to be pulled from the state for this component?

Reply:
No, `adminUrl: wpAdminUrl` is the `adminUrl` prop passed in, but renamed to `wpAdminUrl` for the component to use.